src/commcare_cloud/ansible/roles/postgresql/tasks/set_up_dbs.yml

```diff
@@ -0,0 +1,67 @@
+- name: Create PostgreSQL users
+  become: yes
+  become_user: "{{ db_is_remote and 'ansible' or 'postgres' }}"
+  vars:
+    ansible_ssh_pipelining: true
+  postgresql_user:
+    name: "{{ item.username }}"
+    password: "{{ item.password }}"
+    role_attr_flags: "{{ item.role_attr_flags|default('CREATEDB') }}"
+    state: present
+    port: "{{ postgresql_port }}"
+    login_host: "{{ db_is_remote and postgresql_host or omit }}"
+    login_user: "{{ db_is_remote and postgres_users.root.username or omit }}"
+    login_password: "{{ db_is_remote and postgres_users.root.password or omit }}"
```

Comment:
what's the significance of `omit` in theres?

Reply:
`omit` is an ansible feature we rarely use, but when you need it it's super useful: https://docs.ansible.com/ansible/latest/user_guide/playbooks_filters.html#omitting-parameters.

Basically if something evaluates to

```
<module>:
  arg1: 'value'
  arg2: omit
```
that's the same as if you hadn't mentioned `arg2` at all:
```
<module>:
  arg1: 'value'
```
In this line, I use it because neither `login_password: null` or `login_password: ""` have the same meaning as just not mentioning `login_password` in the first place. In the case of postgresql on an SSHable VM, we run as the `postgres` linux user and do not need `login_username` and `login_password`; in the case of a remote db (RDS), we can't run as a linux user, so we have to provide a `postgresql` user and password (the `root` user above).